browser/resources/settings/email_aliases_page/tests/email_aliases_modal.test.tsx

```diff
@@ -395,4 +402,30 @@ describe('EmailAliasModal', () => {
     })
   }
 
+  it('shows error message when deleting alias fails', async () => {
+    mockEmailAliasesService.deleteAlias = jest.fn().mockImplementation(
+      () => Promise.resolve({ errorMessage: 'emailAliasesDeleteAliasError' }))
+
+    render(
+      <DeleteAliasModal
+        onReturnToMain={mockOnReturnToMain}
+        alias={mockAlias}
+        emailAliasesService={mockEmailAliasesService}
+      />
+    )
+
+    // Click delete button
+    await act(async () => {
+      const deleteButton = screen.getByText('emailAliasesDeleteAliasButton')
+      clickLeoButton(deleteButton)
+    })
```

Comment:
should we check the button is disabled while we're deleting?

Reply:
added a check